source/javascripts/controllers/_WorkflowsController.js.erb

```diff
@@ -646,6 +646,7 @@ angular.module("BitriseWorkflowEditor").controller("WorkflowsController", functi
 		}
 
 		viewModel.selectedStep = step;
+		$scope.$digest();
```

Comment:
needed because of react hook which happens outside of angular context. From inside angular context it won't harm